CMakeLists.txt

```
@@ -301,6 +305,12 @@ if (FLB_SYSTEM_LINUX)
   include(cmake/s390x.cmake)
 endif ()
 
+# Enable signed char support on Linux AARCH64 if specified
+if (FLB_LINUX_ON_AACH64)
```

Oh, there is a typo:

```suggestion
if (FLB_LINUX_ON_AARCH64)
```

---

good catch :)